scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -392,11 +401,26 @@ private int getTotalNumberOfJobs(final DBJobDataParameters params) {
                     queryString.append("and parentId = null ");
                 }
 
+                if (!params.getJobName().isEmpty()) {
+                    queryString.append("and lower(jobName) like :jobName ");
```

Comment:
if you use lower on the database, you should also apply toLowerCase() to the parameter